packages/terra-action-footer/tests/wdio/action-footer-spec.js

```diff
@@ -1,16 +1,11 @@
-const {
-  viewports,
-  withActionsThemeables,
-  withoutActionsThemeables,
-} = require('./common');
+const { viewports } = require('./common');
 
 describe('ActionFooter', () => {
   describe('Empty', () => {
```

Comment:
We are in here restructuring tests, does capturing a screenshot and testing accessibility of an empty action footer add value or can we drop this?

Reply:
Are there any strong opinions for keeping or removing it?

Reply:
Let's remove it.

Reply:
Removed: https://github.com/cerner/terra-core/pull/2021/commits/7300cbb728738327a20cb1423ad56c86284efdc6